sources/packages/backend/apps/db-migrations/src/sql/CRAIncomeVerifications/Standardize-monetary-columns.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE
+    sims.cra_income_verifications
+alter column
+    reported_income type NUMERIC(8, 2);
+
+ALTER TABLE
```

Comment:
As a suggestion to avoid repetition, `ALTER TABLE` can be combined as below.
```sql
ALTER TABLE
    sims.cra_income_verifications
ALTER COLUMN
    reported_income TYPE INT,
ALTER COLUMN
    cra_reported_income TYPE INT;
```

Reply:
Trying to combine multiple alter columns into one alter table results in a syntax error. The proper syntax is to have them all separate.

error: error: syntax error at or near "column"

Reply:
It does work and the same was used in another migration. Please give me a call if needed.